pkg/rpc/convert.go

```diff
@@ -290,6 +291,7 @@ func ConvertToRpcScanResponse(results report.Results, os *ftypes.OS, eosl bool)
 		rpcResults = append(rpcResults, &scanner.Result{
 			Target:          result.Target,
 			Vulnerabilities: ConvertToRpcVulns(result.Vulnerabilities),
+			Type:            os.Family,
```

Comment:
Is there any reason not to use `result.Type`?

Reply:
Ah good catch.